selenium-system-test/src/test/scala/hts/pages/AuthorityWizardPage.scala

```diff
@@ -52,4 +63,37 @@ object AuthorityWizardPage extends WebPage {
   def submit()(implicit driver: WebDriver): Unit =
     find(cssSelector("input.button")).get.underlying.click()
 
+  def setGivenName(givenName: String)(implicit driver: WebDriver): Unit =
+    find(name("itmp.givenName")).get.underlying.sendKeys(givenName)
```

Comment:
avoid `.get` on `Option` as this will throw when the `Option` is not defined. For this case you could do something like this instead:
```scala
find(name("itmp.givenName")).foreach(_.underlying.sendKeys(givenName))
```

Reply:
the same thing applies to the other methods below